components/brave_perf_predictor/browser/third_party_extractor.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_THIRD_PARTY_EXTRACTOR_H_
+#define BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_THIRD_PARTY_EXTRACTOR_H_
+
+#include <string>
+
+#include "base/containers/flat_map.h"
+#include "base/memory/singleton.h"
+#include "base/values.h"
+
+namespace brave_perf_predictor {
+
+class ThirdPartyExtractor {
```

Comment:
After reading both header and source files it still hard to understand the purpose of this class.
Maybe we can rename it to `ThirdPartyEntityProvider`? Because it's main responsibility is not about extracting but rather giving some meta info, basically this is a dictionary.
And also write a 2-3 lines of comments to give some context about it.